keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -199,55 +201,42 @@ admins explicitly by the cluster admin with the use of `Pass` rules.
+total summation of the AdminNetworkPolicyIngressRules and AdminNetworkPolicyEgressRules
+in a single ANP instance, will be 100.
 
 Conflict resolution: Two policies are considered to be conflicting if they are assigned the
 same `priority` and apply to the same resources or a union of resources. In order to avoid such conflicts, 
 we propose to include tooling for ANP resources to help alert the admin to potentially ambiguous ANP
-priority scenarios, more details in [risks and mitigations](#risks-and-mitigations). However, ultimately
-it will be the job of the CNI to decide how to handle overlapping priority situations in the respective 
-implementations. 
-
-For conformance and portability purposes, it will be required that implementations
-support the range of 1000 numeric ANP priorities (0 through 999) per cluster and a maximum of 100 rules
-within an ANP instance. This ensures that policy manifests written using priorities
-0 through 999 are portable across clusters and implementations without the
-need to re-write policy manifests or re-adjust priority allocations and also ensures we maintain 
-safe upper bounds. 
-
-#### Multiple AppliedTos per policy instance
-
-With the current Kubernetes NetworkPolicy v1 api, an instance of a policy can be applied
-to a single set of pods defined using a single selector instance.
-However with the proposed ANP specification it shall be possible to apply the policy to 
-multiple unique sets of Namespaces/pods within a single ANP policy instance, where each such set can 
-be configured with its own set of peer endpoints and policy actions. This enables greater flexibility
-of configuration, and improves ease of use by not requiring cluster admins to have to come up
-with unique priority values for each instance of rule while retaining explicit rule ordering
-within each policy (multiple rules wihin a ANP instance are prioritized by their listing
-order within the ANP manifest). A fully specified new rule (including AppliedTo selector, peer and action
-type definitions) can hence be unambiguously inserted into an existing ANP without requiring any reshuffling 
-of priority values.
+priority scenarios, more details in [risks and mitigation](#risks-and-mitigation). However, ultimately
+it will be the job of the network policy implementation to decide how to handle overlapping priority 
+situations. 
 
 #### Rule Names
 
 In order to help future proof the ANP api, a built in mechanism to identify each
 allow/deny/pass rule is required. Such a mechanism will help administrators organize 
 and identify individual rules within an AdminNetworkPolicy resource.
 We propose to introduce a new string field, called `name`, in each `AdminNetworkPolicy`
-ingress/egress rule. The `name` of a rule must be unique within the same ANP. However,
-two ANP instances could have rules with same names.
+ingress/egress rule. Currently the `name` of a rule is optional and does need to 
```

Comment:
you mean `does` or `does not`? If it does need to be unique within a policy (we will need to specify the scope here), I guess we'll have to say there's no way in the API validation that we can enforce that, and implementations would need to take care of the validation

Reply:
I meant "does not" because there's no way to validate it... I'll try and add a note

Reply:
```
Currently the `name` of a rule is optional and does not need to 
be unique, due to the fact that there's no easy way to validate this in-tree. It 
is suggested that implementations provide a mechanism to validate any name 
duplications.  
```

Reply:
Is something like this alright?

Reply:
Sounds good to me

Reply:
Fixed